src/array.cr

```diff
@@ -919,37 +914,11 @@ class Array(T)
   # a = [1, 2, 3, 4, 5, 6]
   # a.fill(2, 2) { |i| i * i } # => [1, 2, 4, 9, 5, 6]
   # ```
-  def fill(start : Int, count : Int, & : Int32 -> T) : self
-    return self if count <= 0
-
-    start += size if start < 0
-
-    raise IndexError.new unless 0 <= start < size && start + count <= size
-
-    to_unsafe_slice(start, count).fill(offset: start) { |i| yield i }
-
-    self
-  end
-
-  # :ditto:
-  @[Deprecated("Use `#fill(start, count, &)` instead")]
+  @[Deprecated("Use `Indexable::Mutable#fill(start, count, &)` instead")]
```

Comment:
For consistency's sake.

```suggestion
  @[Deprecated("Use `fill(start, count, &)` instead")]
```

Reply:
This is perfectly fine. It shows clearly that the alternative method isn't defined by `Array` itself, unlike the other suggested methods / calls.

Reply:
I missed that, thanks for the clarification 🙏